Merge remote-tracking branch 'bk1285/master'

HeerNMeester opened the pull request:
Adding Dutch 11x10 layout. Renaming Dutch10x9 layout for consistency.

bk1285 commented:
The commit was intended to add the dutch layout, @HeerNMeester ? I'm a bit confused, regarding the diff...

HeerNMeester commented:
Apologies, I'm new to GIT. 
I updated my fork with your master before I added my changes. Somehow all changes by others since I forked are taken into my update as well. I'll will cancel the pull request. Meanwhile, I found a few small bugs in the Dutch 11x10 code during my test run. I'll try to make another request.
